tar_scm.py

```diff
@@ -261,6 +261,9 @@ def detect_version(self, args, repodir):
         def get_timestamp(self, args, repodir):
             d = {"parent_tag": None, "versionformat": "%ct"}
             timestamp = self.detect_version(d, repodir)
+            # avoid future times due to broken commits
+            if datetime.datetime.fromtimestamp(timestamp) > datetime.datetime.now():
+                  timestamp = datetime.datetime.now()
             return int(timestamp)
```

Comment:
This would only work for git. Once @aspiers merged the rest of my nextRev branch, we should make this detection in TarSCM/archive.py. Then its done for all SCM types